### PR TITLE
Add rscan and r2scan parameters to DFTD3 XC parameters

### PR DESCRIPTION
**Pull request by ShigeruIwase-PFN**
# What
Add rscan and r2scan parameters to DFTD3 XC parameters.

# Why
To use rscan/r2scan D3 correction

# Ref
https://pubs.aip.org/aip/jcp/article/154/6/061101/199772/r2SCAN-D4-Dispersion-corrected-meta-generalized

### CODE REVIEW

**Review by dshintani-pfn — APPROVED** [review]
I confirmed the values and damping=bj

File: torch_dftd/dftd3_xc_params.py [hunk 1]
```diff
@@ -328,6 +328,18 @@ def get_dftd3_default_params(
             rs6 = 0.4860
             s18 = 0.0000
             rs18 = 4.5000
+        elif xc == "rscan":
+            rs6 = 0.4702
+            s18 = 1.0886
+            rs18 = 5.7341
+            # Parameters are taken from Table 1 of ref:
+            # https://pubs.aip.org/aip/jcp/article/154/6/061101/199772/r2SCAN-D4-Dispersion-corrected-meta-generalized
```
Comment (ktns) [inline torch_dftd/dftd3_xc_params.py:336]:
```suggestion
            # https://doi.org/10.1063/5.0041008
```

File: torch_dftd/dftd3_xc_params.py [hunk 2]
```diff
@@ -328,6 +328,18 @@ def get_dftd3_default_params(
...
+        elif xc == "r2scan":
+            rs6 = 0.4948
+            s18 = 0.7898
+            rs18 = 5.7308
+            # Parameters are taken from Table 1 of ref:
+            # https://pubs.aip.org/aip/jcp/article/154/6/061101/199772/r2SCAN-D4-Dispersion-corrected-meta-generalized
```
Comment (ktns) [inline torch_dftd/dftd3_xc_params.py:342]:
```suggestion
            # https://doi.org/10.1063/5.0041008
```

**Review by ktns — COMMENTED** [review]
What do you think about using the doi instead?

File: torch_dftd/dftd3_xc_params.py (hunk 1 above)
Comment (ShigeruIwase-PFN) [reply torch_dftd/dftd3_xc_params.py:336]: Nice catch! I agreed.

**Pull request merged**